corehq/apps/app_manager/models.py

```diff
@@ -5152,6 +5152,12 @@ def import_app(app_id_or_source, domain, source_properties=None, validate_source
         del source['build_spec']
     app = cls.from_source(source, domain)
     app.cloudcare_enabled = domain_has_privilege(domain, privileges.CLOUDCARE)
+
+    for module in app.get_modules():
+        if isinstance(module, ReportModule):
+            for report_config in module.report_configs:
+                report_config.uuid = 'copy-' + report_config.uuid
```

Comment:
this is possible - I like keeping the original uuid around, and didn't want to implement logic for ```copy-#-```.  I don't think it's super likely that a user will copy an app enough times for this to become a problem.